docs/.vuepress/config.js

```diff
@@ -0,0 +1,192 @@
+module.exports = {
+  theme: 'cosmos',
+  title: 'Aragon-Chain Documentation',
+  locales: {
+    '/': {
+      lang: 'en-US'
+    },
+  },
+  base: process.env.VUEPRESS_BASE || '/',
+  themeConfig: {
+    repo: 'ChainSafe/aragon-chain',
+    docsRepo: 'ChainSafe/aragon-chain',
```

Comment:
I was never able to get npm to run on my machine. I dont know if there are plans to create an aragon chain repo or if this should be the ethermint repo and the aragon branch